src/platform/site-wide/component-library-analytics-setup.js

```diff
@@ -3,35 +3,178 @@
  * to translate component library actions into analytics dataLayer events.
  */
 import _recordEvent from 'platform/monitoring/record-event';
-import { kebabCase } from 'lodash';
 import { getSectionLabel } from 'applications/static-pages/subscription-creators/subscribeAccordionEvents';
 
 const analyticsEvents = {
-  Modal: [{ action: 'show', event: 'int-modal-show' }],
+  Modal: [{ action: 'show', event: 'int-modal-show', prefix: 'modal' }],
```

Comment:
@pdavies88 - went with `prefix` since I got a lint error on `parameter_prefix`.

> (property) parameter_prefix: string
Identifier 'parameter_prefix' is not in camel case.eslint[camelcase](https://eslint.org/docs/rules/camelcase)

Or should I go with `parameterPrefix`?

Reply:
@bsmartin-ep I think prefix is fine and describes what it is doing